lib/clone_kit/emitters/base_active_record_emitter.rb

```diff
@@ -0,0 +1,22 @@
+module CloneKit
+  module Emitters
+    class BaseActiveRecordEmitter
+      def initialize(model_klass, query: :all)
```

Comment:
Two thoughts here:

1. Part of the assumption (per the readme) is that the package doesn't supply emitters, instead that each app is responsible for writing its own.  I see this emitter along with the mongo one are used in tests though.  To that end, should these be moved to the `spec` folder?
2. If we don't move these and do export them from the gem, this changes the method signature, which only needed `model_klass` previously.  I think it might make more sense to drop the second argument here, and simply have the `scope` method call `klass.all` directly.